Assets/Game/Scripts/Explodables/ExplodableBase.cs

```diff
@@ -75,6 +79,9 @@ protected virtual void OnExplosionComplete() {}
 
         protected virtual void OnDestroy () {
             _collection.Remove(this);
+            GameController.Instance.TacticalViewToggled -= ToggleView;
```

Comment:
Double check the question mark is valid here (it should be fine).

```suggestion
            GameController.Instance?.TacticalViewToggled -= ToggleView;
```

This will fix the bug below. Triggers when the editor is closed.

![image](https://github.com/user-attachments/assets/4931cae0-0c04-42a8-8800-f22f4a773d20)